src/static/css/blog-docs.less

```diff
@@ -422,6 +422,61 @@
     - cfgov-blog-docs
 */
 
+@gutter__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@gutter-half__em: @gutter__em/2;
+
+.expandable__filters {
+    border: 1px solid @gray-50;
+    margin: unit(45px / @base-font-size-px, em) 0 @gutter__em 0;
+
+    &.expandable__padded, .expandable_content {
+        background:  @gray-5;
+    }
+
+    .expandable_content {
+        margin: 0 @gutter__em @gutter__em;
+
+        .respond-to-max(@mobile-max {
+            margin: 0 @gutter-half__em @gutter-half__em;
+        });
+    }
+
+    .expandable_header {
+        padding: @gutter__em/3 @gutter__em;
+
+        .respond-to-max(@mobile-max {
+            padding: @gutter__em/3 @gutter-half__em;
+        });
+    }
+
+    .expandable_header-right {
+        .respond-to-max(@mobile-max {
+            margin-left: unit(20px / @base-font-size-px, em);
+         });
+    }
+
+    .chosen-select {
+      width: 100%;
+    }
+
+    .lt-ie9 {
+        .ie-date_format {
+            display: u-inline-block();
```

Comment:
This should actually be `u-inline-block();` (without the `display: `), otherwise the Less won't be parsed and ends up in the CSS assets (see https://github.com/cfpb/cfgov-refresh/pull/574/files#diff-4a1b733d46ab894d6953df312702af65R7494)